src/test/java/teammates/test/cases/action/InstructorFeedbackEditSaveActionTest.java

```diff
@@ -99,8 +99,8 @@ public void testExecuteAndPostProcess() {
                 + "instr1@course1.tmt|||Updated Feedback Session "
                 + "<span class=\"bold\">(First feedback session)</span> for Course "
                 + "<span class=\"bold\">[idOfTypicalCourse1]</span> created.<br>"
-                + "<span class=\"bold\">From:</span> Wed Feb 01 00:00:00 UTC 2012"
-                + "<span class=\"bold\"> to</span> Thu Jan 01 00:00:00 UTC 2015<br>"
+                + "<span class=\"bold\">From:</span> Tue Jan 31 18:15:00 UTC 2012"
```

Comment:
Similarly here, I suppose because of the test case configuration?

Reply:
Yup same data as above.